docs/content/docs/data-structures/overview.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Overview of Distributed Data Structures
+menuPosition: 1
+---
+
+The Fluid Framework provides developers with distributed data structures (DDSes) that automatically ensure that each
+connected client has access to the same state. The APIs provided by DDSes are designed to be familiar to programmers
+who’ve used common data structures before.
+
+When using a DDS, you can largely treat it as a local object. You can add data to it, remove data, update it, etc.
```

Comment:
There are a lot of caveats with this statement. Related data across DDSs, or even guidance below to split content into multiple keys within same DDS requires people to consider impacts of concurrent edits. I feel we need some write up on that, and a reference to it every time we say it's as simple as local object, as we see even our own engineers continue to make mistakes in this space.

Reply:
I'm responsible for the original wording, and I approve Vlad's message. :)

Reply:
BTW, I'm happy to write article / docs (stand-alone :)), as long as you promise to help to prof-read and integrate into right place :) Especially on concurrency

Reply:
That'd be awesome!